compose.yml

```diff
@@ -26,7 +26,7 @@ services:
 
   # Run with `docker compose run --rm import-sample-data` to import or re-import the sample datasets
   import-sample-data:
-    image: osgeo/gdal:latest
```

Comment:
does this relate to the PR ?

Reply:
It doesn't but the osgeo/gdal:latest image doesn't exist anymore on docker hub.